api/env.docker

```diff
@@ -1,7 +1,7 @@
 PYTHONUNBUFFERED=0
 
 DJANGO_SETTINGS_MODULE=conf.settings
-DJANGO_SECRET_KEY="ny#b__$$f6ry4wy8oxre97&-68u_0lk3gw(z=d40_dxey3zw0v1"
+DJANGO_SECRET_KEY=example_key
```

Comment:
Making it a reminder to change it would be nice too.
```suggestion
DJANGO_SECRET_KEY=example_key_CHANGE_IT_IN_PRODUCTION
```

Reply:
We can also update `settings.py` to raise an error if it's unchanged.

Reply:
Thanks for the suggestion, and I like that idea @dhruvkb! I'll add an issue for it :)